Auth/AuthAPI.py

```diff
@@ -1,7 +1,44 @@
 # Score API here
 from flask import Blueprint
+from flask import request, jsonify, current_app
 import sys
-from db import db
+from db import credentials
+import jwt
 sys.path.append("../")
 
 auth_api = Blueprint("auth", __name__)
+
+@auth_api.route('/register', methods=['POST'])
+def register():
+    username = request.form.get('username')
+    pw = request.form.get('passwordHash')
+    credentials.append({"username": username, "passwordHash": pw})
```

Comment:
A more defensive implementation will be to check if username and passwordHash is "" or None, otherwise will result in successful registration for a empty username/password